src/main/java/use_cases/sales_report/SalesReporterInputData.java

```diff
@@ -97,7 +98,7 @@ public static ArrayList<String> serialNumList(ArrayList<String[]> rows){
      */
     // return the number of items sold when given a list of string arrays in the
     // format [DateTime,Username,Action,Item Name,Quantity,Serial Number] and a string representing an item name.
-    public static int itemSold(ArrayList<String[]> rows, String serialNum){
+    public static int getItemSold(ArrayList<String[]> rows, String serialNum){
```

Comment:
I think all these changes only change the name of the method to make them easy to understand.